src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,28 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_PASS_AGE = 18;
+    private static final int MIN_LENGTH_PASSWORD = 6;
+    private StorageDao storageData = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
```

Comment:
we should also add check for existence user with such login in db

Reply:
fixed